### PR TITLE
Add prompted and survey details to workflow node detail panel

### PR DESCRIPTION
**Pull request by marshmalien**
https://issues.redhat.com/browse/AAP-18754

Screenshot 2024-04-24 at 10 02 18 AM

Screenshot 2024-04-22 at 10 58 32 AM

### CODE REVIEW

File: frontend/awx/resources/templates/WorkflowVisualizer/components/JobTemplateDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+
+function useAggregateJobTemplateDetails({
+  template,
+  node,
+}: {
+  template: JobTemplate;
+  node: GraphNodeData;
+}) {
+  const { launch_data: promptValues, resource: nodeValues } = node;
+  const { data: nodeLabels } = useGet<AwxItemsResponse<ILabel>>(nodeValues?.related?.labels);
```
Comment (nixocio) [inline frontend/awx/resources/templates/WorkflowVisualizer/components/JobTemplateDetails.tsx:37]: There are several requests being made here, do we need to add loading state for them?

File: frontend/awx/resources/templates/WorkflowVisualizer/components/PromptDetail.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  if (children === null || typeof children === 'undefined' || children === '') {
+    return <></>;
+  }
+  if (isEmpty) {
+    return <></>;
+  }
+
+  let tooltip = null;
+  if (isOverridden) {
+    const overriddenText = t(`This prompt value has been overriden. The default value was: `);
```
Comment (nixocio) [inline frontend/awx/resources/templates/WorkflowVisualizer/components/PromptDetail.tsx:32]:
```suggestion
    const overriddenText = t(`This prompt value has been overridden. The default value was: `);
```

File: frontend/awx/resources/templates/WorkflowVisualizer/hooks/useGetNodeTypeDetail.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { useTranslation } from 'react-i18next';
+import { UnifiedJobType } from '../types';
+
+export function useGetNodeTypeDetail(type?: UnifiedJobType) {
+  const { t } = useTranslation();
+  const typeMapping = {
+    job: t('Job template'),
+    workflow_job: t('Workflow job template'),
+    project_update: t('Project '),
```
Comment (nixocio) [inline frontend/awx/resources/templates/WorkflowVisualizer/hooks/useGetNodeTypeDetail.tsx:9]:
```suggestion
    project_update: t('Project'),
```

**Comment by jerabekjiri** [comment]
Creating/editing node doesn't update the `variables` (extra_vars) in the wizard.
 
[screen-recorder-tue-apr-30-2024-14-29-33.webm](https://github.com/ansible/ansible-ui/assets/19647757/3e610307-71d9-4abd-ab34-325cc9347837)

File: frontend/awx/resources/templates/WorkflowVisualizer/components/JobTemplateDetails.tsx (hunk 1 above)
Comment (marshmalien) [reply frontend/awx/resources/templates/WorkflowVisualizer/components/JobTemplateDetails.tsx:37]: Resolving. We chatted about this and decided it wasn't ideal to show a spinner for a subsection of the details. We could make the entire panel show a spinner but I would prefer to handle that in a separate issue.

**Review by jerabekjiri — APPROVED** [review]
LGTM, great work :+1: :tada:

**Pull request merged**